apps/jubmoji-quest/src/pages/quests/[id].tsx

```diff
@@ -62,8 +118,14 @@ export default function QuestDetailPage() {
             </Link>
           );
         })}
-        <LeaderBoard></LeaderBoard>
-        <Button variant="secondary">Update team score</Button>
+        {quest.proofType === $Enums.ProofType.TEAM_LEADERBOARD && (
```

Comment:
Nice, this feels clean. So only quests that have team leaderboard will show a leaderboard, and their power will also be related to teams? Like do we have some sort of way of funneling the top 3 teams into a claimable power?

Reply:
I mean, you could do this entirely on the frontend, just based on the fetched leaderboard if the user has a team pubkey in the top 3 the power is displayed